domain/src/main/java/org/sagebionetworks/research/domain/form/InputField.java

```diff
@@ -80,8 +83,7 @@
      * @return data type for this input field. The data type can have an associated ui hint
      */
     @NonNull
-    @InputDataType
-    String getFormDataType();
+    InputDataType getFormDataType();
```

Comment:
The InputDataType on the iOS side is a one of type with a couple of options Base and Collection are 2 of about 4 options. Rather enumerating out all of the Strings for each type I made this class. I figure it will be easier to add new types through this class than by having to figure out which strings to add.